convergent_born_iteration/benchmark_solve2d.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python3
+"""
+A script to measure the solver's time-efficiency using the example found in example_solve2d.py.
+"""
+import timeit
+
+import jax.numpy as jnp
+
+from convergent_born_iteration import log, electro_solver, example_solve2d
+
+log.getChild(__name__)
+
+def main():
+    log.debug(f'Starting {__name__} ...')
+
+    log.info('Defining the problem.')
+    grid, k0, permittivity, current_density, _ = example_solve2d.define_problem([480, 640])
+
+    log.info(f'Converting problem of shape {grid.shape} to JAX.')
+    permittivity = jnp.array(permittivity)
+    current_density = jnp.array(current_density)
+
+    log.info('Solving repeatedly...')
+    times = timeit.repeat(
+        lambda: electro_solver.solve(grid, k0, permittivity, current_density, maxiter=1000, tol=0).block_until_ready(),
```

Comment:
You jitted the function in the other file; typically we only jit on the highest level, i.e. in this file.